### PR TITLE
GH-90 endpoint to get transactions from pool concerning account

### LINKED ISSUE #90: endpoint to get transactions from pool concerning account
- [x] GET `/tx_pool/:account:` returning all transactions having from_acc or to_acc
- [x] return empty list if none found

### PR DESCRIPTION
**Pull request by d-velev**
closes #90

### CODE REVIEW

File: apps/aehttpclient/lib/client.ex [hunk 1]
```diff
@@ -41,10 +49,12 @@ defmodule Aehttpclient.Client do
           :info ->
             response = Poison.decode!(body, keys: :atoms!)
             {:ok, response}
-          :peers ->
```
Comment (thepiwo) [inline apps/aehttpclient/lib/client.ex]: why is `:peers` and `:balance` removed?

Comment (d-velev) [reply apps/aehttpclient/lib/client.ex]: :peers and :balance will be handled by the default json_response()

Comment (thepiwo) [reply apps/aehttpclient/lib/client.ex]: great

**Comment by thepiwo** [comment]
can you remove the compiler warnings as well?

**Comment by thepiwo** [comment]
can you add a test as well for getting an existinng tx from the pool and one where there is none transaction?
